src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -91,20 +92,33 @@ public function supportsDenormalization($data, $type, $format = null)
 
     /**
      * {@inheritdoc}
-     *
-     * @throws InvalidArgumentException
      */
-    public function denormalize($data, $class, $format = null, array $context = [])
+    protected function setObjectToPopulate($data, string $class, array &$context)
     {
-        // Avoid issues with proxies if we populated the object
-        if (isset($data['@id']) && !isset($context[self::OBJECT_TO_POPULATE])) {
-            if (isset($context['api_allow_update']) && true !== $context['api_allow_update']) {
-                throw new InvalidArgumentException('Update is not allowed for this operation.');
+        try {
+            parent::setObjectToPopulate($data, $class, $context);
```

Comment:
Can you add a `return;` here and move the code in the `catch` part outside?

Reply:
you mean an empty catch right?

Reply:
done